app/views/api/v0/followers/users.json.jbuilder

```diff
@@ -1,6 +1,7 @@
 json.array! @follows do |follow|
-  json.type_of "user_follower"
-  json.id      follow.id
+  json.type_of    "user_follower"
+  json.id         follow.id
+  json.created_at follow.created_at
```

Comment:
please stick to ISO/RFC3339 format, see other endpoints. You can use 

https://github.com/forem/forem/blob/14416f79cc02f002e34f53339ed28a27adee1a48/app/helpers/articles_helper.rb#L56-L58